code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -1752,22 +1752,66 @@ shaved
 	icon_state = "ears_plain"
 	species_allowed = list(SPECIES_TAJ)
 
-/datum/sprite_accessory/hair/taj_ears_clean
-	name = "Tajaran Clean"
-	icon_state = "hair_clean"
-	species_allowed = list(SPECIES_TAJ)
-
 /datum/sprite_accessory/hair/taj_ears_bangs
 	name = "Tajaran Bangs"
 	icon_state = "hair_bangs"
 	species_allowed = list(SPECIES_TAJ)
 
+/datum/sprite_accessory/hair/taj_ears_bangs_alt
+	name = "Tajaran Bangs (Alt)"
+	icon_state = "hair_bangs_alt"
+	species_allowed = list(SPECIES_TAJ)
+
+/datum/sprite_accessory/hair/taj_ears_short_fringe
+	name = "Tajaran Short Fringe"
+	icon_state = "hair_shortfringe"
+	species_allowed = list(SPECIES_TAJ)

```

Comment:
Nit: You could use inheritance to do something like:
````
/datum/sprite_accessory/hair/taj
	name = "Tajaran Bangs (Alt)"
	icon_state = "hair_bangs_alt"
	species_allowed = list(SPECIES_TAJ)

/datum/sprite_accessory/hair/taj/ears_short_fringe
	name = "Tajaran Short Fringe"
	icon_state = "hair_shortfringe"
````
Then redo `taj_` to `taj/` and not have to re-set `species_allowed`.